contracts/src/core/EigenDAAccessControl.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {AccessControl} from "@openzeppelin/contracts/access/AccessControl.sol";
+import {AccessControlConstants} from "src/core/libraries/v3/access-control/AccessControlConstants.sol";
+
+contract EigenDAAccessControl is AccessControl {
```

Comment:
Document that this is meant to be deployed as the single centralized source of truth ACL for all EigenDA contracts (which will need to call into this contract), as opposed to the traditional OZ way (iiuc) of having each contract inherit AccessControl directly and own its own ACL.